assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -68,47 +69,38 @@ const sizeFromWidth = (
   };
 };
 
-// we want to set the size of story pages based on the available space
-// this means we need to take the window.innerWidth value and remove the built in WP nav and the dashboard nav according to breakpoints
-const getTrueInnerWidth = () => {
-  const { innerWidth } = window;
-  if (innerWidth >= theme.breakpoint.raw.tablet) {
-    return innerWidth - WP_LEFT_NAV_WIDTH - DASHBOARD_LEFT_NAV_WIDTH;
-  } else if (innerWidth < VIEWPORT_WP_LEFT_NAV_HIDES) {
-    return innerWidth;
-  } else {
-    return innerWidth - WP_LEFT_NAV_WIDTH;
-  }
-};
-
 export default function usePagePreviewSize(options = {}) {
   const { thumbnailMode = false, isGrid } = options;
+  // When the dashboard is pulled out of wordpress this id will need to be updated.
+  // For now, we need to grab wordpress instead because of how the app's rendered
+  const dashboardContainerRef = useRef(document.getElementById(WPBODY_ID));
```

Comment:
This will be a nice way to quickly adjust that.

Reply:
that's what i was thinking too, i'm glad this makes sense